_docker/backup/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM developer.redhat.com/ruby:2.0.0
+
+ARG http_proxy
+ARG https_proxy
+
+COPY mariadb/MariaDB.repo /etc/yum.repos.d/MariaDB.repo
+
+RUN yum erase -y git
+RUN curl -s https://setup.ius.io/ | bash
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | bash
```

Comment:
These should probably be above the first yum, then you can inline the two yum commands lastly do a yum clean all